packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -7,6 +7,8 @@ import 'dart:typed_data';
 
 import 'package:dwds/data/build_result.dart';
 import 'package:dwds/dwds.dart';
+import 'package:html/dom.dart';
```

Comment:
We weren't actually using this package before, it was just transitive. Unfortunately adding new direct dependencies may slow down out eventually null safety migration, so I'd only like to do so if we know that it will be migrated/maintained.

@natebosch is this something that is planned for `package:html` eventually?

Alternatively you could parse it out with a regex (Yeah I know...)

Reply:
Yes, this will be migrated soon. I'm not sure the long term future for the package, but at least in the short term we will be maintaining it.

Reply:
Okay SGTM,

in that case @mdebbar remove the `# transitive ...` dependency tag from the pubspec.yaml and move it over with the rest of the direct dependencies

Reply:
👍